config/secrets.defaults.yml

```diff
@@ -14,3 +14,9 @@ test:
     secret_key: IBM_CLOUD_OBJECT_STORAGE_SECRET_KEY
   ibm_cloud_object_storage:
     <<: *ibm_cloud_object_storage_defaults
+  iks_defaults: &iks_defaults
+    api_key: IBM_CLOUD_IKS_API_KEY
+    hostname: IBM_CLOUD_IKS_HOSTNAME
+    port: IBM_CLOUD_IKS_PORT
```

Comment:
@nasark `port` has to be an integer: https://app.travis-ci.com/github/ManageIQ/manageiq-providers-ibm_cloud/jobs/537165737#L2152-L2178
It can be anything, `1234` for example